linkerd/app/src/env.rs

```diff
@@ -477,6 +479,20 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
         let mut require_identity_for_inbound_ports =
             parse(strings, ENV_INBOUND_PORTS_REQUIRE_IDENTITY, parse_port_set)?.unwrap_or_default();
 
+        let allowed_ips = {
+            let ips = parse(strings, ENV_INBOUND_IPS, parse_socket_addr_set)?.unwrap_or_default();
```

Comment:
we don't want to parse socket addrs here

```
Invalid configuration: invalid environment variable
[     0.000554s] ERROR ThreadId(01) linkerd_app::env: Not a valid address: 10.42.0.3
[     0.000570s] ERROR ThreadId(01) linkerd_app::env: LINKERD2_PROXY_INBOUND_IPS="10.42.0.3" is not valid: AddrError(MissingPort)
```

Reply:
Ohh, got it --- I misinterpreted the issue! Will fix!

Reply:
fixed in b1954e6dc37b56f205b8e589114f2d2b3cdab070